Tools/scripts/build_binaries.sh

```diff
@@ -230,20 +230,21 @@ build_arduplane() {
         return
     }
     skip_build $tag $ddir || {
-        for v in v1 v2 v3 v4 v4pro; do
-            skip_board_waf px4-$v && continue
-            echo "Building plane for px4-$v in $BUILDROOT"
+        for v in px4-v1 px4-v2 px4-v3 px4-v4 px4-v4pro; do
+            skip_board_waf $v && continue
+            echo "Building plane for $v in $BUILDROOT"
             waf configure --board $b --out $BUILDROOT clean && waf plane || {
```

Comment:
I do not get this change, you add px4- prefix and later on (line 241) remove it.  Why?